scripts/fix_rename.py

```
@@ -0,0 +1,141 @@
+            value = getattr(fti, attr, None)
+            if value and OLD_PREFIX in value:
+                new_value = value.replace(OLD_PREFIX, NEW_PREFIX)
+                setattr(fti, attr, new_value)
+                changed.append((type_id, attr, value, new_value))
+    return changed
+
+
+def invalidate_schema_cache(changed):
+    try:
+        from plone.dexterity.schema import SCHEMA_CACHE
+    except Exception as exc:  # pragma: no cover
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)